ocfweb/component/banner.py

```diff
@@ -0,0 +1,23 @@
+from collections import namedtuple
+from urllib.request import urlopen
+
+import yaml
+
+
+class BannerMessage(namedtuple('BannerMessage', [
+    'visible',
+    'force_lab_closed',
+    'html',
+])):
+    pass
+
+
+def get_banner_message():
+    """Get the front page banner message from the default location."""
+    with urlopen('file:///home/s/st/staff/banner.yaml') as f:
+        tree = yaml.load(f)
```

Comment:
`yaml.safe_load` please

pyyaml is idiotic. `yaml.load` is stupidly dangerous and will execute Python code embedded in the YAML (it should be called `dangerous_load` instead). You almost always want `safe_load` instead.